ac.soton.eventb.emf.inclusion.generator/src/ac/soton/eventb/emf/inclusion/generator/InclusionAdaptor.java

```diff
@@ -48,7 +49,7 @@
  *
  */
 
-public class InclusionAdaptor  implements IAdapter {
+public class InclusionAdaptor extends EventBTranslatorAdapter implements IAdapter {
 

```

Comment:
Why override methods without changing? The ones in EventBTranslatorAdapter are more up to date. Maybe declare EventBTranslatorAdapter directly and delete InclusionAdapter.

Reply:
Because we copied it initially ...